pages/index.tsx

```diff
@@ -0,0 +1,537 @@
+import { FunctionComponent } from 'react';
+
+import PageLink, { CallToAction } from '../components/PageLink';
+
+import AnalyticsManagerIcon from '../images/svg/analyticsmanager.svg';
+import DevIcon from '../images/svg/dev.svg';
+
+import StickyBox from 'react-sticky-box';
+
+import styles from './index.module.scss';
+
+function emptyRowElement() {
+  return <div style={{ display: 'flex', padding: '15px' }} />;
+}
+
+const LandingPage: FunctionComponent = () => {
+  const tocContent = (
+    <div>
+      {' '}
+      <p>
+        <a href="#stage1" className={styles.tocItem}>
+          1. Tracking plan management – no engineering required
+        </a>
+      </p>
+      <p>
+        <a href="#stage2" className={styles.tocItem}>
+          2. Tracking plan management with validation – one time SDK install
+          required
+        </a>
+      </p>
+      <p>
+        <a href="#stage3" className={styles.tocItem}>
+          3. End-to-end tracking plan collaboration and implementation workflows
+        </a>
+      </p>
```

Comment:
Can we use an `<ol>` element here to make this format as lists, with the numbers aligned, and the content for each point aligned with indent?

Reply:
I'm not sure about that, wanted to have 3 steps emphasized by those numbers. Maybe this is better
<img width="329" alt="Screen Shot 2021-04-28 at 15 18 42" src="https://user-images.githubusercontent.com/9194227/116402511-2dcbdc00-a835-11eb-8d03-d2c093258f83.png">